guide/principles/principles.md

```diff
@@ -0,0 +1,168 @@
+
+-->
+
+# Bitcoin design principles
+
+As a new technology, Bitcoin offers the opportunity of a decentralized open financial system, where participants share the role of securing the network. This is important to give everyone equal and direct access without needing intermediaries. To make this a reality, we encourage everyone working on products to deliberately support the core principles of designing for Bitcoin. These are principles we in the Bitcoin Design Community have identified and stand behind. Some of these come from the technology itself, and others from the behaviour and ethos of the community. Although every use case and product is different, applications should strive to follow these principles. Diverging from them should only be done with very good reason.
+
+
+- [Decentralization](#decentralization)
+- [Inclusion](#inclusion)
+- [Transparency](#transparency)
+- [Self-custody](#self-custody)
+- [Security](#security)
+- [Privacy](#privacy)
+- [Interoperability](#interoperability)
+
+---
+
+## Decentralization
+
+> Encourage users to run their own Bitcoin node and connect directly to the network
+
+Unlike traditional banking systems, the Bitcoin economy does not require new users to seek permission from anyone. Bitcoin has no central point of control. No one person or entity is in charge. Connecting to any [node]({{ '/guide/glossary/#node/' | relative_url }}) on the network gives you the same rights and responsibilities, which also ensures that there is no single point of failure.
+
+**Do**
+- Manage your own [private keys]({{ '/guide/private-key-management/introduction/' | relative_url }})
+- Run a full Bitcoin node when possible
+- Alternatively, use a light client with the [p2p network]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki'}}) using [compact block filters]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki'}}) 
+- Offer the user choice of what node and other external services to connect to
+
+**Don't**
+- Introduce a single point of failure between the user and the Bitcoin network
+- Build products that stop working if the project shuts down
+
+---
+
+## Inclusion
+
+> Provide equal access regardless of the user's location, language, ability and prior knowledge of Bitcoin
+
+There are no background checks, credit checks, or gatekeepers to Bitcoin. A Kenyan farmer has the same access to Bitcoin as a Wall Street trader. 
+
+And while Bitcoin is already widely used by a large number of people, it pales in comparison with the many more that are likely to use it in the future. We need to design products that are prepared for people new to Bitcoin, without reducing the possibilities. This means using plain and familiar language, explaining things in the context where they are needed, not overwhelming people with technical detail, and more.
+
+**Do**
+- Provide equal and direct access to the Bitcoin network, without middlemen and exclusive requirements
```

Comment:
> Provide equal and direct access to the Bitcoin network, without middlemen and exclusive requirements

I'd try to break this down somehow and simplify it, I'm not sure it's clear to a newbie what we mean by this?

Reply:
Addressed in 661e72f